packages/browser/src/tracing/request.ts

```diff
@@ -136,14 +153,16 @@ export function instrumentOutgoingRequests(client: Client, _options?: Partial<Re
       return event;
     });
 
-    addFetchEndInstrumentationHandler(handlerData => {
-      if (handlerData.response) {
-        const span = responseToSpanId.get(handlerData.response);
-        if (span && handlerData.endTimestamp) {
-          spanIdToEndTimestamp.set(span, handlerData.endTimestamp);
+    if (trackFetchStreamPerformance) {
+      addFetchEndInstrumentationHandler(handlerData => {
+        if (handlerData.response) {
+          const span = responseToSpanId.get(handlerData.response);
+          if (span && handlerData.endTimestamp) {
+            spanIdToEndTimestamp.set(span, handlerData.endTimestamp);
+          }
```

Comment:
Instead of passing this bool around everywhere, would it make sense to instead just not call addFetchEndInstrumentation handler? I would also add a comment somewhere saying why we are defaulting to not instrumenting it.

Reply:
yeah I think I pushed another commit doing just that while you were reviewing, sorry!